### DISCUSSION TITLE
Dockerized NPM protect with Crowdsec

### DISCUSSION THREAD

**Discussion by mapime** [NginxProxyManager/nginx-proxy-manager#4085] [Q&A]
Hello,

i have trouble to protect my dockerized NPM on a Debian LXC with Crowdsec.
At the moment i have enrolled the crowdsec with the official command: 
"sudo cscli console enroll -e context XXXXXXXX" 

on the LXC i have Docker with Compose and the official NPM from here.
Additionally added the volume:
"- /var/log/nginx:/var/log/nginx"
but no access logs come here inside.

Can you help me, how is the best effect to protect my LXC and NPM?

thanks so much

**Comment by RadioHerzblut** [comment]
Hier suche ich auch noch eine funktionierende Lösung. Hast Du schon eine ?

**Comment by mapime** [comment]
Leider nein. Wäre immer noch offen für eine Lösung! 
Danke!

**Comment by RadioHerzblut** [comment]
Hallo Maxpilz. 
Ich bin vom NPN weg. Ich habe keine Lösung gefunden.
Ich betreibe nun den Pangolin Reverse Proxy.  
Bei der Installation kannst du Crowdsec aktivieren.
Pangolin ist auf einen 4€ VPS installiert. Der VPS baut eine Wiregard Verbindung in mein LAN auf. Im LAN habe ich das Gegenstück NEWT in Portainer am laufen.

Nachteil : ein externer VPS der Kosten erzeugt. 
Vorteil : An meinen Router sind keine Port auf




Habe dieses Setup seit Monaten. Läuft absolut stabil.

Anbei eine gute Anleitung:

https://youtu.be/z3Ao9CWH0GU?si=E5Iz_n4Sq6C_6jmE
